.eslintrc.js

```diff
@@ -56,6 +56,8 @@ module.exports = {
     'no-unused-vars': 'off', // Pour ne pas avoir l'erreur en doublon
     'prefer-destructuring': ['error', { VariableDeclarator: { object: true, array: false } }],
     'function-paren-newline': ['error', 'consistent'],
+    'react/jsx-uses-react': 'off',
+    'react/react-in-jsx-scope': 'off',
```

Comment:
A quoi serve ces deux regles ? Qu'est-ce qui t'a fait choisir de ne pas les activer ?

Reply:
c'est ce qui est expliqué ici : https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html il faut les désactiver pour que React 17 JSX transform fonctionne. Sinon eslint nous demande d'importer React sur chaque fichier